modules/ossm-attach-workloads-to-control-plane.adoc

```diff
@@ -0,0 +1,35 @@
+:_mod-docs-content-type: Concept
+[id="attach-workloads-to-control-plane"]
+= Attach workloads to the control plane
+:context: ossm-attach-workloads-to-control-plane
+
+When {istio} is configured to use the `InPlace` strategy, the `IstioRevision` resource that the Operator creates always has the same name as the {istio} resource. To attach the workloads to a control plane deployed using the `InPlace` strategy, you can set the `istio.io/rev` namespace label to the name of the `IstioRevision`. 
+
+.Example istio.io namespace label to attach workloads to a mesh
+[source,terminal]
+----
+oc label namespace <namespace-name> istio.io/rev=<revision-name>
```

Comment:
should you include the $ sign here at the start? I  think that is the standard practice

Reply:
done